tools/lib.sh

```diff
@@ -75,3 +75,7 @@ function contains_element() {
   return 1
 }
 
+function validate_config() {
+  log "Validating json config file $1"
+  python -m json.tool $1 1>/dev/null 
```

Comment:
I looked into this a little further. Because of the plugin architecture, changing this would require quite a bit of refactoring. This is probably outside the scope of this ticket, and should be in its own issue.

In the mean time, should I just remove the python code to validate the config file?